### DISCUSSION TITLE
Getting Sponsors

### DISCUSSION THREAD

**Discussion by IP80808080** [community/community#123531] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I'm seeking advice on the best platform or method to approach potential sponsors for our project. Could someone please provide recommendations or share insights based on their experience?

**Comment by hafiz-muhammad** [comment]
GitHub has a sponsorship program: [https://github.com/sponsors](https://github.com/sponsors)

**Comment by anjuan** [comment]
Hi @IP80808080, I'm the Engineering Manager for GitHub Sponsors. It's exciting that you're looking for sponsors for your project. Can you describe the community you've already developed around your project?

**Comment by IP80808080** [comment]
Thank you for reaching out! I'm thrilled to discuss potential sponsorship opportunities for our blockchain-based project.

As a college student, I've been collaborating with my team members and mentor to develop our project. While our community is still growing, we're dedicated to fostering an inclusive and supportive environment where contributors of all backgrounds can collaborate and innovate together.
